src/com/hms/flexyosisoftconnector/OSIsoftConfig.java

```diff
@@ -93,6 +93,9 @@
   /** default http timeout in seconds for if it is not set through configuration file. */
   private static final int HTTP_TIMEOUT_SECONDS_DEFAULT = 2;
 
+  /** Default logging level for the SC logger library is set to 4 for 'info'. */
+  private static final int LOGGING_LEVEL_DEFAULT = 4;

```

Comment:
Is there a reason for this change? I think the logging should only really output warnings at most by default. If additional information is needed, it would be set via config file.

Reply:
Having a sane default simplifies the configuration process. You might be right that Warning is a better default than info. I can update this change.